### PR TITLE
chore: fix wasm dir issue

### PR DESCRIPTION
**Pull request by vishal-kanna**
closes: #1122

### CODE REVIEW

**Comment by vishal-kanna** [comment]
I think addition of this code might fix the dir issue @verabehr

File: cmd/palomad/root.go [hunk 1]
```diff
@@ -33,7 +34,7 @@ func NewRootCmd() *cobra.Command {
 	// set Bech32 address configuration
 	params.SetAddressConfig()
 
-	tempApp := palomaapp.New(cosmoslog.NewNopLogger(), db.NewMemDB(), io.MultiWriter(), true, db.OptionsMap{})
+	tempApp := palomaapp.New(cosmoslog.NewNopLogger(), db.NewMemDB(), io.MultiWriter(), true, simtestutil.NewAppOptionsWithFlagHome(palomaapp.DefaultNodeHome))
```
Comment (byte-bandit) [inline cmd/palomad/root.go:37]: Why are you using the `simtestutil` package outside of simulation? Why not pass proper options?

Comment (taariq) [reply cmd/palomad/root.go:37]: @vishal-kanna can we resolve now this so we can merge into the next release, please?

Comment (vishal-kanna) [reply cmd/palomad/root.go:37]: Okay @taariq

**Comment by taariq** [comment]
@byte-bandit let's merge on your confirm.

**Pull request merged**